libraries/AP_NavEKF3/AP_NavEKF3.h

```diff
@@ -455,6 +455,7 @@ class NavEKF3 {
     // enum for processing options
     enum class Options {
         JammingExpected     = (1<<0),
+        PlaneRequiresGPS    = (1<<1),
```

Comment:
Nothing about this is `Plane`-specific AFAICS.

Reply:
Correct me if I'm wrong, but I think it is Plane-specific implicitly.

The check blocking initialization only triggers if assume_zero_sideslip() is true:
 https://github.com/ArduPilot/ardupilot/blob/145cc4bb260c56fb92632886a624bc06b56b0e4d/libraries/AP_NavEKF3/AP_NavEKF3_core.cpp#L476
 
 `assume_zero_sidslip` is only true if `get_fly_forward()` is true & the vehicle is not a ground vehicle.
 https://github.com/ArduPilot/ardupilot/blob/145cc4bb260c56fb92632886a624bc06b56b0e4d/libraries/AP_NavEKF3/AP_NavEKF3_Control.cpp#L665-L671
 
AFAICT only the Rover (a ground vehicle) & Plane (in non-VTOL mode) ever return `get_fly_forward()` - `true`.
 
Obviously if another non-ground vehicle w/ fly_forward `true` gets added this check would apply to it as well.